spec/controllers/amr_heatmap_controller_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'rails_helper'
+
+RSpec.describe AmrHeatmapController, type: :controller do
+  create_users
+
+  # Admin context
+  context 'Admin user' do
+    # create users
+    before do
+      sign_in @admin
+    end
+
+    describe "GET index" do
+      it "sees correct AMR data" do
+        # Create a test sample (in a project, as required) that contains Amr data.
+        # Note that pipeline_runs_data needs to have two attributes in order for the sample
+        # to be available for the AmrHeatmapController: a job_status equal to the constant
+        # STATUS_CHECKED as defined in the PipelineRun class, and an output_states array
+        # containing an OutputState for amr_counts with state equal to the constant
+        # STATUS_LOADED as defined in the PipelineRun class.
+        create(:project, users: [@admin, @joe], samples_data: [{
+                 pipeline_runs_data: [{
+                   amr_counts_data: [{
+                     gene: "IamA_Gene"
+                   }],
+                   job_status: PipelineRun::STATUS_CHECKED,
+                   output_states_data: [{
+                     output: "amr_counts",
+                     state: PipelineRun::STATUS_LOADED
+                   }]
+                 }]
+               }])
+
+        # Now that the sample has been created, grab it from the test db
+        power = Power.new(@admin) # for safe access
```

Comment:
I'm not sure if using Power is the best way to get the newly created sample/amr_counts (though it does work). I believe `create()` returns the object it created. Will defer to @tfrcarvalho

Reply:
Done! Removed usage of Power from tests and created sample separately.